istioctl/cmd/testdata/authz/converter/service-role-with-constraints.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: "rbac.istio.io/v1alpha1"
```

Comment:
When testing this file manually, I found the following generated in stdout in addition to the error message in stderr:
```
map[destination.ip:[10.1.2.3 10.2.0.0/16]]
```
We should not generate anything to stdout when an error happens, and ideally the above message should be generated to stderr as well.

Reply:
I have a print statement that I forgot to remove :)